### DISCUSSION TITLE
Inputs character count

### DISCUSSION THREAD

**Discussion by MedanWeb** [filamentphp/filament#10349] [Ideas]
It would be cool to add a character count to the input field. For example, add a ->countable() method (or something like that) to TextInput and Textarea. And make it in AlpineJs, example: [https://jsfiddle.net/alexjustesen/97va6d1h/](url) so that you can count without adding ->reactive() / ->lazy() to the component, reducing the number of requests and, accordingly, the load. Sorry for bad English.

**Comment by ShaungBhone** [comment]
check it [here](https://v2.filamentphp.com/tricks/count-how-many-characters-left-to-type-in-the-field).

**Reply by joseph-d** [reply]
It's a solution, yes, but it would be nicer if this could be accomplished without the network request (ie, not using reactive or lazy) as @amfetaminf2017 said

**Comment by bohdan-shulha** [comment]
Any updates on this? Will the PR be accepted for the feature if I create one?
